yt_dlp/extractor/twitter.py

```diff
@@ -1654,6 +1693,7 @@ def _real_extract(self, url):
         info['title'] = broadcast.get('status') or info.get('title')
         info['uploader_id'] = broadcast.get('twitter_username') or info.get('uploader_id')
         info['uploader_url'] = format_field(broadcast, 'twitter_username', 'https://twitter.com/%s', default=None)
+        info['display_id'] = display_id
         if info['live_status'] == 'is_upcoming':
             return info
```

Comment:
Kinda unrelated, can we do this too? I forgot to add this a while ago and it's been bothering me
```suggestion
        if info['live_status'] == 'is_upcoming':
            self.raise_no_formats('This live broadcast has not yet started', expected=True)
            return info
```